msteams-platform/task-modules-and-cards/cards/people-picker-adaptive-card.md

```diff
@@ -0,0 +1,247 @@
+
+#### Data.Query
+
+|Property |Type |Required |Description |
+|--|--|--|--|
+|**dataset** |String |Yes |The type of data that must be fetched dynamically.|   
+
+#### dataset
+The following table provides predefined values as **dataset** for people picker:   
+
+|dataset|Search Scope
+|--|--|
+|**graph.microsoft.com/users** |Search all members across the organization.|
+|**graph.microsoft.com/users?scope=currentContext** |Search within the members of the current conversation, such as chat or channel in which the particular card is sent.|      
+
+> [!NOTE]
+> Currently, the search functionality for all the members across the organization is supported in 1:1 chats with bots, Adaptive Card task modules and tabs only. It is not supported in other 1:1 chats, group chats or channels.  
+
+### Example
+The code example for creating People Picker with organization search is as follows:
+
+```json 
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "TextBlock",
+            "size": "Medium",
+            "weight": "Bolder",
+            "text": "People Picker with Org search enabled"
+        },
+        {
+            "type": "Input.ChoiceSet",
+            "choices": [],
+            "choices.data": {
+                "type": "Data.Query",
+                "dataset": "graph.microsoft.com/users"
+            },
+            "id": "people-picker",
+            "isMultiSelect": true
+        },
+        {
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "People Picker with Org search enabled"
+    },
+    {
+      "type": "Input.ChoiceSet",
+      "choices": [],
+      "choices.data": {
+        "type": "Data.Query",
+        "dataset": "graph.microsoft.com/users"
+      },
+      "id": "people-picker",
+      "isMultiSelect": true
+    }
+  ],
+  "actions": [
+    {
+      "type": "Action.Submit",
+      "title": "Submit"
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.2"
+}
+```  
+
+The following image illustrates People Picker in Adaptive Cards with organization search:
+
+![People Picker Org Search](../../assets/images/cards/peoplepicker-org-search.png)
+
+To enable search within a list of conversation members, use the appropriate dataset defined in the [dataset](#dataset) table. `isMultiSelect` property is used to enable the selection of multiple users  in the control. It's set to false by default and this setting allows you to select single user only.
+
+### Data Submission
+
+You can `Submit Action` in an Adaptive Card. You can provide the inputs in the card that are sent back to the bot through an `invoke`. The `Submit Action` is similar to other `Input control` in Adaptive Cards. The `invoke` payload consists of a list of input IDs to their corresponding values.  
```

Comment:
> The `Submit Action` is similar to other `Input control` in Adaptive Cards.
This is a bit vague in here - Action is a better term to use instead of Input Control. Also, we can use `Action.Submit` or `Action.Execute` to submit data from people picker. cc: @shiladityasaha